### ISSUE TITLE
DeDRM v7.1.0: Ultimately failed to decrypt

### ISSUE THREAD

**Issue by anuragfromindia** [apprenticeharper/DeDRM_tools#1613]
## CheckList

- [x] `The Title` and The `Log Title` are setted correctly.
- [x] Clarified about `my environment`.
- [x] Code block is used for `the log`.

---

## Log
<details><summary>Here's the log</summary>

```log
Gather KFX-ZIP (from KFX Input) 1.37.0: Importing C:/Users/anura/Documents/My Kindle Content/B084SRMDTZ_EBOK/B084SRMDTZ_EBOK.azw
Gather KFX-ZIP (from KFX Input): Gathered 3 file(s) as C:\Users\anura\AppData\Local\Temp\calibre__mvvxzop\8k627r95.kfx-zip
DeDRM v7.1.0: Trying to decrypt 8k627r95.kfx-zip
Using Library AlfCrypto DLL/DYLIB/SO
Using Library AlfCrypto DLL/DYLIB/SO
Decrypting KFX-ZIP ebook: 8k627r95
Found 1 keys to try after 0.1 seconds
Decrypting KFX DRM voucher: amzn1.drm-voucher.v1.a4c39530-bd01-4754-b791-b158c41c80c0.voucher
DeDRM v7.1.0: Failed to decrypt with error: Failed to decrypt KFX DRM voucher with any key
DeDRM v7.1.0: Looking for new default Kindle Key after 0.1 seconds
searching for kinfoFiles in C:\Users\anura\AppData\Local
Found K4PC 1.25+ kinf2018 file: C:\Users\anura\AppData\Local\Amazon\Kindle\storage\.kinf2018
Decrypted key file using IDString '2761528885' and UserName 'Director'
DeDRM v7.1.0: Found 1 new key
Decrypting KFX-ZIP ebook: 8k627r95
Got DSN key from database default_key_1
Found 1 keys to try after 3.0 seconds
Decrypting KFX DRM voucher: amzn1.drm-voucher.v1.a4c39530-bd01-4754-b791-b158c41c80c0.voucher
DeDRM v7.1.0: Ultimately failed to decrypt after 3.0 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Package KFX (from KFX Input) 1.37.0: Packaging C:\Users\anura\AppData\Local\Temp\calibre__mvvxzop\8k627r95.kfx-zip
Processing container: B084SRMDTZ_EBOK.azw
KFX metadata reader (from KFX Input) activated for C:\Users\anura\AppData\Local\Temp\calibre__mvvxzop\fbfij7r9_add_books\0\B084SRMDTZ_EBOK.kfx-zip
Added Kohinoor/ कोहिनूर (Hindi Edition) to db in: 0.2
Added 1 books in 3.6 seconds
```
</details>

**Comment by ElleKayEm** [comment]
That book isn't from Kindle for PC 1.17.  Were you unable to download it without updating?  Are you able to download it with Kindle for PC 1.26 or earlier?

**Comment by anuragfromindia** [comment]
I love you! Thanks for pointing out the Kindle version issue. 

Initially, I installed Kindle 1.17, but I couldn't open the book as it prompted me to update the software. I clicked on ok, thinking I'll have to update it manually. 

Now that you have mentioned it's not the same version that I was assuming it to be; it turns out that clicking the 'ok' button on that prompt automatically updated the kindle, which I overlooked. 

Then I tried manually installing Kindle v1.26, and it worked! Thank you so much!
